src/main/java/com/minecolonies/coremod/commands/NavigationMenuType.java

```diff
@@ -0,0 +1,82 @@
+package com.minecolonies.coremod.commands;
+
+import org.jetbrains.annotations.NotNull;
+
+public enum NavigationMenuType implements MenuType
+{
+    Colonies(new NavigationMenu("colonies",
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00115)